src/components/common/FieldGroup/FieldGroup.tsx

```
@@ -0,0 +1,30 @@
+  return <p>{title}</p>;
+}
+
+function TitleWrapper({ children }: { children: ReactNode }) {
+  return <div className={styles.titleWrapper}>{children}</div>;
+}
+
+function Label({ text, hidden }: { text: string; hidden?: boolean }) {
+  return !hidden && <label className={styles.fieldLabel}>{text}</label>;
+}
+
+function Error({ message }: { message: string }) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)